docs(@nguniversal/builders): Added Builder Readme Overview (#2478)
angular/universal: modules/builders/README.md

| ssl           | boolean (default: false)    | Serve using HTTPS.                                                                                                                                     |          |
| sslKey        | string                      | SSL key to use for serving HTTPS.                                                                                                                      |          |
| sslCert       | string                      | SSL certificate to use for serving HTTPS.                                                                                                              |          |
| proxyConfig   | string                      | Proxy configuration file.                                                                                                                              |          |

## Static Generator

### Description

This builder is used to prerender pages of your application using the experimental [Clover](../common/clover/README.md) implementation.
Prerendering is the process where a dynamic page is processed at build time generating static HTML.

Attention:

> Use this if you don't have any server side Code - as "serverTarget" is not configurable.

Author: Is that correct?

# Angular Universal Builders

Angular Universal comes with some custom Builders. All Parameters are also specified in their schema.json file.  

> In any case this Readme.md gets outdated - it's worthy to have a look there!

Reviewer: ```suggestion
In any case this Readme.md gets outdated - it's worthy to have a look there!
```

Lets remove this note and make sure to keep the doc updated instead.

# Builders

- prerender
- ssr-dev-server
- static-generator

## Prerender

> ### Summary

Reviewer: ```suggestion
### Description
```

> This Builder will prerender the routes that you define in a Route-File or as Parameter.

Reviewer: ```suggestion
This builder is used to prerender pages of your application. Prerendering is the process where a dynamic page is processed at build time generating static HTML.

For more information about this see: https://angular.io/guide/prerendering

```

Use this if you want to serve fast Pages, where you don't have changing Content on _each request_.

> ### Parameter

Reviewer: ```suggestion
### Options
```

| ------------- | -------------------------------------------- | --------------------------------------------------------------------------------- | -------- |
| browserTarget | Pattern: `^[^:\s]+:[^:\s]+(:[^\s]+)?$`       | Target to build.                                                                  | ✔️       |
| serverTarget  | Pattern: `^[^:\s]+:[^:\s]+(:[^\s]+)?$`       | Server target to use for prerendering the app.                                    | ✔️       |
| routesFile    | A Filename - Attention no schema validation! | The path to a file containing routes separated by newlines.                       |          |
| routes        | Unique Strings                               | The routes to render.                                                             |          |
| guessRoutes   | Bolean (default: true)                       | Whether or not the builder should extract routes and guess which paths to render. |          |
| numProcesses  | Number (min: 1, default: 1)                  | The number of cpus to use. Defaults to all but one.                               |          |

> ### Example

Reviewer: ```suggestion

```

> WIP

Reviewer: ```suggestion
=
```

## SSR Dev Server

### Summary

> This builder is the replacement for ng serve, if you develop an Angular Universal App.

Reviewer: ```suggestion
This builder is the replacement for ng serve, if you develop an Angular Universal application.
```

> This builder is the replacement for ng serve, if you develop an Angular Universal App.

Reviewer: ```suggestion

```
This is not quite right, this builder isn't intended to replace `ng serve` but rather offer a way to serve and watch a universal application.

> Use this if you want to develop locally an Angular Universal Application

Reviewer: ```suggestion
This builder will launch a development server with live-reload capabilities for Angular Universal applications. 
```

> It will hot reload only things that you have changed and trigger a refresh in the browser.

Reviewer: ```suggestion

```

| inspect          | boolean (default: false)    | Launch the development server in inspector mode and listen on address and port '127.0.0.1:9229'.                                                       |          |
| ssl              | boolean (default: false)    | Serve using HTTPS.                                                                                                                                     |          |
| sslKey           | string                      | SSL key to use for serving HTTPS.                                                                                                                      |          |
| sslCert          | string                      | SSL certificate to use for serving HTTPS.                                                                                                              |          |
| proxyConfig      | string                      | Proxy configuration file.                                                                                                                              |          |

> | waitBeforeReload | number                      | Waiting time before reload the page (in Milliseconds)                                                                                                  |          |

Reviewer: ```suggestion

```

> ### Example

Reviewer: ```suggestion

```

> WIP

Reviewer: ```suggestion

```

## Static Generator

> ### Summary (WIP)

Reviewer: ```suggestion
### Description
```

> Builds the browser and server, then renders each route in options.routes and writes them to prerender/<route>/index.html for each output path in the browser result.  

Reviewer: ```suggestion
This builder is used to prerender pages of your application using the experimental [Clover](../common/clover/README.md) implementation. Prerendering is the process where a dynamic page is processed at build time generating static HTML.

Attention: 
```

Use this if you don't have any server side Code - as "serverTarget" is not configurable.

> ### Parameter

Reviewer: ```suggestion
### Option
```

| Name          | Type of Value                                | Description                                                                       | Required |
| ------------- | -------------------------------------------- | --------------------------------------------------------------------------------- | -------- |
| browserTarget | Pattern: `^[^:\s]+:[^:\s]+(:[^\s]+)?$`       | Target to build.                                                                  | ✔️       |

> | routesFile    | A Filename - Attention no schema validation! | The path to a file containing routes separated by newlines.                       |          |

Reviewer: ```suggestion
| routesFile    | string | The path to a file containing routes separated by newlines.                       |          |
```

There are paths validation in code.

| routes        | array of Unique Strings                      | The routes to render.                                                             |          |

> | guessRoutes   | Bolean (default: true)                       | Whether or not the builder should extract routes and guess which paths to render. |          |

Reviewer: ```suggestion
| guessRoutes   | Boolean (default: true)                       | Whether or not the builder should extract routes and guess which paths to render. |          |
```

> | numProcesses  | Number (min: 1, default: 1)                  | The number of cpus to use. Defaults to all but one.                               |          |

Reviewer: ```suggestion
| numProcesses  | number (min: 1, default: 1)                  | The number of cpus to use. Defaults to all but one.                               |          |
```
Not consistence.

> ### Example

Reviewer: ```suggestion

```

> WIP

Reviewer: ```suggestion

```
